### PR TITLE
Desktop version portfolio

### PR DESCRIPTION
**Pull request by abreham1222**
This is the figma design for the desktop version portfolio design which is web responsive design.  This project contained:-
1. added index.html
2. added style.css
3. added important image
4. added an updated readme file
5. added mit.md license

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -57,17 +65,17 @@ <h2 class="header-subtitle">I’m a Software Developer</h2>
...
-      <div class="image-p">
-        <img class="image-placeholder" src="Images/skeloton/placeholder.png" alt="gymfit picture with beutifull girl">
-        <div class="multi">
+       <div class="work-container">
+         <h2 class="recent-work">My Recent Works</h2>
+         <hr class="re-line">
+       </div>
+       <div class="image-p">
+         <img class="image-placeholder" src="Images/skeloton/placeholder.png" alt="gymfit picture with beutifull girl">
+         <div class="multi">
```
Comment (Kamba56) [inline index.html:78]: - [x] Kindly style the contents of this container to match the design template. You are required to stick to the design as much as possible, kindly fix the spacings to match the design
## Your design
r1
## Figma template
s1
.

File: index.html [hunk 2]
```diff
@@ -79,24 +87,24 @@ <h2 class="multi-post">Multi-Post Stories</h2>
...
         </div>
-        <ul class="pro-tag">
+        <ul class="pro-tag" id="img1-tag">
           <li class="pro-h">html</li>
           <li class="pro-h">bootstrap</li>
           <li class="pro-h">Ruby</li>
         </ul>
-           <button class="see-proj" type="button">See Project</button>
+           <button class="see-proj" id="see-btn" type="button">See Project</button>
       </div>
```
Comment (Kamba56) [inline index.html:106]: - [x] Kindly fix the spacing of the buttons and technologies for the works section to match the Figma template, please.
## Your design
r2
## Figma template
s2

File: index.html [hunk 3]
```diff
@@ -216,11 +225,15 @@ <h5 class="text-white">Skills</h5>
             </div>
         </section>
       <form action="https://formspree.io/f/xlezlkyw" method="post">
-        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      <div class="forms">  
+        <p class="mine">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      <div class="inputs">
```
Comment (Kamba56) [inline index.html]: - [x] Kindly style the form to match the design template please.
## Your design
r3
## Figma template
s3

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @abreham1222, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
AlmostThereGIF

## Highlights
- [x]  All linters passed
- [x] Descriptive PR title and description
- [x] Correct use of GitHUb flow

## Required Changes ♻️
_Please check the comments under the review for suggested changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [Optional] It's better to follow commit message standards by making them capitalized and in the imperative form (Example: 'Modify readme file' instead of 'modified readme'). This way, your commit history will be more readable and more professional ✨

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mgmediaweb) [reply index.html:78]: I kindly advise you to follow the previous reviewer's advice.

The title of the section is not the same font, and the language tags do not adapt to the size of the text or are not centered, (I recommend you not to use fixed widths, instead just use padding)

File: index.html (hunk 2 above)
Comment (mgmediaweb) [reply index.html:106]: I kindly advise you to follow the previous reviewer's advice.

File: index.html (hunk 3 above)
Comment (mgmediaweb) [reply index.html]: I kindly advise you to follow the previous reviewer's advice.

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @abreham1222 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

work2

 ## Highlights 💯 
- I realize that you have done a great job doing your project ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by abreham1222** [comment]
Hi @mgmediaweb,
Thank you for your feedback. kindly can you watch my video which is recorded from my laptop which is my code fits with my screen but I do not understand why it is rejected. here is the link
https://www.awesomescreenshot.com/video/10737682

File: index.html [hunk 4]
```diff
@@ -216,11 +225,15 @@ <h5 class="text-white">Skills</h5>
...
         <input name="full-name" type="text" placeholder="Full name" maxlength="30" required>
         <input name="email" type="email" placeholder="Email address" required>
         <textarea name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
         <button class="get" type="submit">Get in touch</button>
+      </div>
```
Comment (rukundoeric) [inline index.html:235]: - [ ] <b>REQUIRED:</b> Like mentioned by the previous reviewer, you need to make sure that your form looks more like the provided design. Firstname and Lastname should be in two different field and also Kindly make sure to style them like how they are styled in Figma design. Also your button text is overlapping. 

| Figma      | Yours |
  | ----------- | ----------- |
| 80047697-0c3f6280-8506-11ea-9416-a4f1800e2329 | Screenshot |

Comment (rukundoeric) [inline index.html]: - [ ] <b>REQUIRED:</b> Like mentioned by the previous reviewer, you need to make sure that your form looks more like the provided design. Firstname and Lastname should be in two different field and also Kindly make sure to style them like how they are styled in Figma design. Also your button text is overlapping. 

| Figma      | Yours |
  | ----------- | ----------- |
| 80047697-0c3f6280-8506-11ea-9416-a4f1800e2329 | Screenshot |

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hello @abreham1222 👋,

_This is an additional review as a result of the second opinion request._

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._


## Required Changes ♻️
_Check the comments under the review._


Keep rocking 🚀!

File: index.html [hunk 5]
```diff
@@ -216,11 +225,15 @@ <h5 class="text-white">Skills</h5>
...
+    </div>
         <hr class="line">
       </form>
```
Comment (ichala) [inline index.html]: - [ ] As mentioned by the previous reviewer, you still didn't fix the form design to match Figma , kindly fix the design to match Figma (update the form inputs design: add padding and update borders and you should be fine)

- _**your design :**_ 

image

- **_Figma Design_**


image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @abreham1222 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

-  No linters Errors ✅ 

 ## Required Changes ♻️

- [ ] **_Check the comments below_**

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 6]
```diff
@@ -416,3 +420,292 @@ textarea {
...
+    padding: 130px 50px;
+    height: 256px;
+  }
+
+  input[type="text"],
+  input[type="email"] {
+    margin-left: 10px;
+    margin-right: 24px;
+    width: 50vw;
+  }
```
Comment (rukundoeric) [inline style.css]: - [ ] <b>OPTIONAL:</b> Before you merge, you could try these and see how it looks: 
    - add these styles in that input:
       ```css
              height: 40px;
              background-color: white;
              border-bottom: 1px solid lightgray;
              outline: none;
              padding-left: 10px;
        ```

File: style.css [hunk 7]
```diff
@@ -416,3 +420,292 @@ textarea {
...
+    height: 114px;
+  }
+
+  .get {
+    margin-left: 10px;
+    padding: 12px 10px;
+    width: 131px;
+    height: 48px;
+  }
+
```
Comment (rukundoeric) [inline style.css]: - [ ] <b>OPTIONAL:</b> Also, add there in here: 

```css
    border: 0;
    text-transform: capitalize;

**Review by rukundoeric — APPROVED** [review]
Hello  @abreham1222  👋,

_This is an additional review as a result of the second opinion request._

It seems you haven't been able to make it exactly like how it is in the Figma design, however, since you did your best, we will go ahead and approve it.  We want students to practice more, that's why we ask to make sure that your implementation looks exactly like the Figma provided design, since thats how its done in Frontend and Full-stack software development positions(when your boss gives you a design, you are expected to implement EXACTLY like the design).

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] check the comments under review

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please note that since we will approve it manually so this could take few hours!

Keep rocking 🚀!

**Comment by abreham1222** [comment]
HI @rukundoeric,
Thank you for your feedback on how can I submit my dashboard shows as incomplete.

**Pull request merged**